pretrain.py

```diff
@@ -3,7 +3,14 @@
 import random
 from urllib.parse import urlparse
 
-from autowebcompat import network, utils
+from autowebcompat.network import *
```

Comment:
I'd prefer the more explicit option (`from autowebcompat import network, utils`), where you know where the functions and variables are coming from.